config/routes.rb

```diff
@@ -4,5 +4,6 @@
 
   resources :collections do
     resources :manifests, only: [:create, :destroy]
+    resources :workspaces, only: [:new, :create, :show]
```

Comment:
Perhaps only `new` and `create` should be nested under `collections` (to easily give a `/collections/:colletion_id/workspaces(/new)` route (that inherently has access to the collection resource) and the `show` (and shortly `index`) would not be nested and you could show or list workspaces outside of the context of a collection.